pkg/controller/workload/workload.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Remove the header here.  Technically we don't have a license out here yet (although we should).  I would say MIT is probably what we should go with.